www/src/components/footer/footer.astro

```diff
@@ -41,51 +41,6 @@ const isLanding = path === "/";
           </div>
         </a>
       </div>
-      <div>
-        <ul
-          class="grid grid-cols-2 gap-6 sm:flex items-center justify-center flex-col sm:flex-row text-center lg:space-x-8"
-        >
-          <li>
-            <a
-              rel="noopener noreferrer"
-              href="/en/introduction"
-              class="hover:dark:text-t3-purple-200">Docs</a
-            >
-          </li>
-          <li>
-            <a
-              rel="noopener noreferrer"
-              href="/en/faqs"
-              class="hover:dark:text-t3-purple-200">FAQ</a
-            >
-          </li>
-          <li>
-            <a
-              rel="noopener noreferrer"
-              href="https://github.com/t3-oss/create-t3-app"
-              class="hover:dark:text-t3-purple-200">Github</a
-            >
-          </li>
-          <li>
-            <a
-              rel="noopener noreferrer"
-              href="https://init.tips"
-              class="hover:dark:text-t3-purple-200">T3 Stack</a
-            >
-          </li>
-        </ul>
-      </div>
-      <a
-        href="https://vercel.com/?utm_source=t3-oss&utm_campaign=oss"
-        rel="noopener noreferrer"
-        target="_blank"
-      >
-        <img
```

Comment:
Did you mean to remove this?

Reply:
Fixed, meant to delete ct3a logo, deleted vercel instead